teamcity/flake8_plugin.py

```diff
@@ -1,74 +1,28 @@
-try:
-    import pycodestyle as pep8
-except ImportError:
-    import pep8
+from flake8.formatting import base
 
 import re
 
 from teamcity.messages import TeamcityServiceMessages
-from teamcity import __version__, is_running_under_teamcity
```

Comment:
It's always been possible to do that. I have no clue why the old version was written the way it was.